services/ResumeService.ts

```diff
@@ -0,0 +1,24 @@
+import { ResumeModel } from 'models/ResumeModel';
+import { NotFoundError } from 'routing-controllers';
+import { Service } from 'typedi';
+import { EntityManager } from 'typeorm';
+import { InjectManager } from 'typeorm-typedi-extensions';
+import Repositories, { TransactionsManager } from '../repositories';
+// import { ResumeRepository } from '../repositories/ResumeRepository';
```

Comment:
nit: is there a reason we need this import?

Reply:
Yeah let's definitely remove any commented-out code

Reply:
Was using it at some point and thought i needed it later, gonna remove it now